frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -844,6 +844,15 @@ function ReaderFooter:addToMainMenu(menu_items)
                     self:refreshFooter(true, true)
                 end,
             },
+            {
+                text = _("Lock tap status bar"),
```

Comment:
Could you update the PR with that text?
```suggestion
                text = _("Lock status bar"),
```